Runtime/Scripts/RGBotConfigs/DefaultActions/DrawLineToAction.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RegressionGames.RGBotConfigs.DefaultActions
+{
+    
+    /**
+     * This action uses Gizmos to draw a line from a given agent to a position in a scene.
+     * Takes the following parameters:
+     *  If drawing a line (either new or old)
+     *    - name (string): And identifier for the line
+     *    - position (Vector3): The position to draw the line to (from the agent)
+     *    - color (Color): The color of the line to draw
+     *  If removing the line:
+     *    - name (string): The line to remove
+     *    - remove (boolean): Set to true
+     *  If removing all lines:
+     *    - removeAll (bool): Set to true
+     */
+    public class DrawLineToAction: RGAction
+    {
+
+        private Dictionary<string, Tuple<Vector3, Color>> _lines = new ();
+
+        public override string GetActionName()
+        {
+            return "DrawLineTo";
+        }
+
+        public override void StartAction(Dictionary<string, object> input)
+        {
+            
+            if ((bool) input.GetValueOrDefault("removeAll", false))
+            {
+                _lines.Clear();
+                return;
+            }
+
+            var lineName = (string) input["name"];
+            if ((bool) input.GetValueOrDefault("remove", false))
+            {
+                _lines.Remove(lineName);
+            }
+            else
+            {
+                var targetPosition = (Vector3) input["position"];
```

Comment:
This seems error prone.  We are getting the target position in `StartAction` and then continuing to render this indicator until they tell us to remove it, but the line's start and end point aren't updating each frame.

We definitely should have this be able to accept and `entityId` as the target so that the line can keep moving both its start and end from frame to frame

Reply:
This is correct and was intentional based on my current use case. Just a heads up though, the line's start **is** updated every frame, since it's based on the agent's origin. I was thinking that a different function or parameter would be provided if you wanted to follow a certain transform.

Reply:
Fixed/new code